x-pack/plugins/security_solution/server/endpoint/routes/resolver/entity/utils/supported_schemas.ts

```diff
@@ -6,6 +6,7 @@
  */
 
 import _ from 'lodash';
+import { allowedExperimentalValues } from '../../../../../../common';
```

Comment:
Same as above, I guess we can use our hook instead of getting the value directly. Also, is this taking the updated value from config? It seems it only takes what is set on the `experimental_features.ts` file but not updated with config, right?

Reply:
Sorry, you cannot use the hook here, just realised this is server code. I guess we can get it from server context instead

Reply:
I am afraid we cannot use the hook on serverside ;/ As for the config, I am not sure, what is the difference between these two? Thanks!

Reply:
My concern here is it is not taking the override value from config. We can get it from route context instead. That would mean sending it down to this schema utils